### PR TITLE
Add scroll in tutorial section of the software

### PR DESCRIPTION
**Pull request by vishalmry**
Issue #71

### CODE REVIEW

File: goa/Classes/menu/LevelHelpOverlay.cpp [hunk 1]
```diff
@@ -70,8 +70,23 @@ void LevelHelpOverlay::gotoGame(Ref* pSender, cocos2d::ui::Widget::TouchEventTyp
...
+                /* 
+                 * To resize the text widget according to the text height
+                 * an additional label is used with the same width, font 
+                 * and font size as the text widget.
+                */
+                auto labelText = Label::createWithSystemFont(_helpText, "arial", 75);
+                labelText->setDimensions(2000, 0);
+                Size contentSizeLabel = labelText->getContentSize();
+                _text->setContentSize(contentSizeLabel);
+                listviewScroll->requestDoLayout();
```
Comment (ankitk20) [inline goa/Classes/menu/LevelHelpOverlay.cpp]: Can't we put this common line of code inside if-else outside of it?

File: goa/Classes/menu/LevelHelpScene.cpp [hunk 2]
```diff
@@ -464,10 +478,25 @@ void LevelHelpScene::gotoGame(cocos2d::Ref *pSender, cocos2d::ui::Widget::TouchE
...
             {
                 _text->setString(LangUtil::getInstance()->translateString(_helpText));
+                /*
+                 * To resize the text widget according to the text height
+                 * an additional label is used with the same width, font
+                 * and font size as the text widget.
+                */
+                auto labelText = Label::createWithSystemFont(_helpText, "arial", 75);
+                labelText->setDimensions(2000, 0);
+                Size contentSizeLabel = labelText->getContentSize();
```
Comment (ankitk20) [inline goa/Classes/menu/LevelHelpScene.cpp]: Can we put this common 4 lines outside of _if-else_?

File: goa/Classes/menu/LevelHelpScene.h [hunk 3]
```diff
@@ -17,6 +17,7 @@ class LevelHelpScene: public cocos2d::Node {
     static cocos2d::Scene *createScene(std::string gameName);
     static LevelHelpScene *create(std::string gameName);
 	cocos2d::ui::Button *_resumeButton;
```
Comment (ankitk20) [inline goa/Classes/menu/LevelHelpScene.h:19]: Align this line

**Review by ankitk20 — CHANGES_REQUESTED** [review]

File: goa/Classes/menu/LevelHelpScene.h (hunk 3 above)
Comment (vishalmry) [reply goa/Classes/menu/LevelHelpScene.h:19]: It is aligned, github is showing it like this.

**Review by ankitk20 — APPROVED** [review]

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**